applications/nrf_desktop/doc/hid_state.rst

```diff
@@ -170,7 +170,7 @@ The ``button_event`` is the source of this type of data.
 
 To indicate a change to this input data, overwrite the value that is already stored.
 
-Since keys on the board can be associated to a usage ID, and thus be part of different HID reports, the first step is to identify to which report the key belongs and what usage it represents.
+Since keys on the board can be associated to a usage ID, and thus be part of different HID reports, the first step is to identify to which report the key belongs to and what usage it represents.
```

Comment:
Aren't there now two "to" particles associated with "belongs". I would remove the one between "identify" and "which".